src/components/views/onboarding/Onboarding.view.tsx

```diff
@@ -48,7 +48,9 @@ enum LabelStatus {
 const OnboardHeader: FC<IOnboard> = ({ step }) => {
 	return (
 		<OnboardHeaderConatiner>
-			<H5 weight={700}>Complete your profile</H5>
+			<H5 style={{ textAlign: 'center' }} weight={700}>
+				Complete your profile
+			</H5>
```

Comment:
use styled-components for styling instead of adding inline styles.

Reply:
Thanks a lot for the review @MohammadPCh  🤩
I fixed it.